src/Validation/Validation.php

```diff
@@ -164,11 +164,11 @@ public static function lengthBetween($check, int $min, int $max): bool
      */
     public static function cc($check, $type = 'fast', $deep = false, $regex = null): bool
     {
-        if (!is_scalar($check)) {
+        if (!is_numeric($check)) {
```

Comment:
This should not be changed. `is_numeric('0000-0000-0000-0000')` doesn't return true.

Reply:
I would be more careful to change basic and stable validation code this drastically.
Lots of false positive "invalidations" because of this suddenly for a lot of apps.

Reply:
This could be `is_string() || is_int()`.

Reply:
> This should not be changed. is_numeric('0000-0000-0000-0000') doesn't return true.

This mistake wouldn't have happened if there were tests cases for this format :)